crypto/src/key/rschnorr/mod.rs

```diff
@@ -155,6 +162,105 @@ impl Derivable for MLRistrettoPrivateKey {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct MLPrivateKey {
```

Comment:
I would call this Secp256k1PrivateKey. The reason we used ML in the other one was because I didn't find a better wrapper name. Embarrassing... I know.